packages/typography/src/Text/Text.tsx

```diff
@@ -1,5 +1,59 @@
-function Text() {
-  return <p className="text-amber-400">Text</p>;
-}
+import clsx from "clsx";
+import React from "react";
+
+import {
+  alignMap,
+  colorMap,
+  sizeMap,
+  weightMap,
+} from "../helpers/prop-mappings";
+
+type TextProps = React.HTMLAttributes<HTMLElement> & {
+  as:
+    | "p"
+    | "span"
+    | "div"
+    | "label"
+    | "em"
+    | "strong"
+    | "b"
+    | "i"
+    | "pre"
+    | "code";
+  align?: keyof typeof alignMap;
+  size?: keyof typeof sizeMap;
+  color?: keyof typeof colorMap;
+  weight?: keyof typeof weightMap;
+};
+
+type TextRef = HTMLElement;
+
+const Text = React.forwardRef<TextRef, TextProps>(
+  (
+    {
+      color = "black",
```

Comment:
Do we want to allow for using other colors from the design system, e.g. a lighter shade of gray or do we always want to restrict it to the ones defined in colormap?

Reply:
We want to restrict it to those defined in the color map